x-pack/test/rule_registry/spaces_only/tests/basic/index.ts

```diff
@@ -18,8 +18,5 @@ export default ({ loadTestFile, getService }: FtrProviderContext): void => {
     after(async () => {
       await deleteSpaces(getService);
     });
-
-    // Basic
-    loadTestFile(require.resolve('./bootstrap'));
```

Comment:
removed this test that tests that no index is bootstrapped when using the reader because the index should already exist.